modules/apps/data-engine/data-engine-taglib/src/main/java/com/liferay/data/engine/taglib/internal/servlet/taglib/util/DataLayoutTaglibUtil.java

```diff
@@ -628,39 +626,42 @@ private String _resolveModuleName(DDMFormFieldType ddmFormFieldType) {
 	}
 
 	private void _setFieldIndexTypeNone(JSONObject jsonObject) {
-		_stream(
-			"rows",
-			rowJSONObject -> _stream(
-				"fields", null, _stream(rowJSONObject.getJSONArray("columns"))),
-			_stream(jsonObject.getJSONArray("pages"))
-		).filter(
-			fieldJSONObject -> Objects.equals(
-				fieldJSONObject.getString("fieldName"), "indexType")
-		).findFirst(
-		).ifPresent(
-			fieldJSONObject -> fieldJSONObject.put("value", "none")
-		);
-	}
+		JSONArray pageJSONArray = jsonObject.getJSONArray("pages");
 
-	private Stream<JSONObject> _stream(JSONArray jsonArray) {
-		return StreamSupport.stream(jsonArray.spliterator(), true);
-	}
+		for (int i = 0; i < pageJSONArray.length(); i++) {
```

Comment:
@JanisZhang Is it necessary to use `for (index)` instead of `for (X x : xs)`? I don't see usage of these indexes except getting the value.

Reply:
@dantewang, I think I can use `for (JSONObject jsonObject : (Iterable<JSONObject>)jsonArray)` . Thanks!